jobbergate-api/alembic/env.py

```diff
@@ -1,13 +1,16 @@
 from logging.config import fileConfig
 
-from alembic import context
 from sqlalchemy import engine_from_config, pool
 
-from jobbergate_api.storage import build_db_url
+from alembic import context
 from jobbergate_api.apps.applications import models  # noqa # must be imported for metadata to work
 from jobbergate_api.apps.job_scripts import models  # noqa # must be imported for metadata to work
 from jobbergate_api.apps.job_submissions import models  # noqa # must be imported for metadata to work
+from jobbergate_api.apps.models import Base
+from jobbergate_api.apps.smart_templates import models  # noqa # must be imported for metadata to work
+from jobbergate_api.apps.job_script_templates import models  # noqa # must be imported for metadata to work
```

Comment:
as far as I could understand by looking at the `Makefile`, the linter only runs against the `jobbergate_api` folder, which means there's no need to add `# noqa` here